var/spack/repos/builtin/packages/subversion/package.py

```diff
@@ -91,6 +95,15 @@ def configure_args(self):
         if '+perl' in spec:
             args.append('PERL={0}'.format(spec['perl'].command.path))
 
+        if '+nls' in spec:
+            args.extend([
+                'LDFLAGS=-L{0}'.format(spec['gettext'].prefix.lib),
+                'LIBS=-lintl',
```

Comment:
```suggestion
                'LIBS=' + spec['gettext'].libs.link_flags,
```
This will automatically determine the appropriate flags to use, although it will also pull in `-lgettextlib` and other libs we might not need. This might not actually be the best idea, so I'll leave this one up to you. There are other `libs` properties available too.

Reply:
So at first glance I wanted to do this anyway, but decided to take a look at the sizes of the libs it pulls in, and it seems like `libintl` is much smaller than the rest (hundreds of kilobytes vs multiple megabytes), which makes me want to restrict it to just that one. To wit:
```
> find /home/cosmicexplorer/tools/s1/opt/spack/linux-alpine3-zen3/gcc-11.2.1/gettext-0.21-4ac6mag7bf2d4wg67wr5a6gcpaq6lz5n/lib -name '*.so*' -type f | parallel -n50 du -h -s
1.7M    /home/cosmicexplorer/tools/s1/opt/spack/linux-alpine3-zen3/gcc-11.2.1/gettext-0.21-4ac6mag7bf2d4wg67wr5a6gcpaq6lz5n/lib/libgettextlib-0.21.so
80K     /home/cosmicexplorer/tools/s1/opt/spack/linux-alpine3-zen3/gcc-11.2.1/gettext-0.21-4ac6mag7bf2d4wg67wr5a6gcpaq6lz5n/lib/libasprintf.so.0.0.0
1.3M    /home/cosmicexplorer/tools/s1/opt/spack/linux-alpine3-zen3/gcc-11.2.1/gettext-0.21-4ac6mag7bf2d4wg67wr5a6gcpaq6lz5n/lib/libgettextsrc-0.21.so
2.5M    /home/cosmicexplorer/tools/s1/opt/spack/linux-alpine3-zen3/gcc-11.2.1/gettext-0.21-4ac6mag7bf2d4wg67wr5a6gcpaq6lz5n/lib/libtextstyle.so.0.1.1
196K    /home/cosmicexplorer/tools/s1/opt/spack/linux-alpine3-zen3/gcc-11.2.1/gettext-0.21-4ac6mag7bf2d4wg67wr5a6gcpaq6lz5n/lib/libintl.so.8.2.0
1.5M    /home/cosmicexplorer/tools/s1/opt/spack/linux-alpine3-zen3/gcc-11.2.1/gettext-0.21-4ac6mag7bf2d4wg67wr5a6gcpaq6lz5n/lib/libgettextpo.so.0.5.7
> spack load subversion+nls
> ls $(which svn)
-rwxr-xr-x 1 cosmicexplorer cosmicexplorer 1.6M May  9 17:55 /home/cosmicexplorer/tools/s1/opt/spack/linux-alpine3-zen3/gcc-11.2.1/subversion-1.14.1-r36gfaufu7pdi2ccsoda3cafvimlengj/bin/svn*
> find /home/cosmicexplorer/tools/s1/opt/spack/linux-alpine3-zen3/gcc-11.2.1/subversion-1.14.1-r36gfaufu7pdi2ccsoda3cafvimlengj/lib -name '*.so*' -type f | parallel -n50 du -h -s
1.1M    /home/cosmicexplorer/tools/s1/opt/spack/linux-alpine3-zen3/gcc-11.2.1/subversion-1.14.1-r36gfaufu7pdi2ccsoda3cafvimlengj/lib/libsvn_repos-1.so.0.0.0
232K    /home/cosmicexplorer/tools/s1/opt/spack/linux-alpine3-zen3/gcc-11.2.1/subversion-1.14.1-r36gfaufu7pdi2ccsoda3cafvimlengj/lib/libsvn_fs-1.so.0.0.0
1.1M    /home/cosmicexplorer/tools/s1/opt/spack/linux-alpine3-zen3/gcc-11.2.1/subversion-1.14.1-r36gfaufu7pdi2ccsoda3cafvimlengj/lib/libsvn_ra_serf-1.so.0.0.0
56K     /home/cosmicexplorer/tools/s1/opt/spack/linux-alpine3-zen3/gcc-11.2.1/subversion-1.14.1-r36gfaufu7pdi2ccsoda3cafvimlengj/lib/libsvn_fs_util-1.so.0.0.0
2.8M    /home/cosmicexplorer/tools/s1/opt/spack/linux-alpine3-zen3/gcc-11.2.1/subversion-1.14.1-r36gfaufu7pdi2ccsoda3cafvimlengj/lib/libsvn_client-1.so.0.0.0
632K    /home/cosmicexplorer/tools/s1/opt/spack/linux-alpine3-zen3/gcc-11.2.1/subversion-1.14.1-r36gfaufu7pdi2ccsoda3cafvimlengj/lib/libsvn_ra_svn-1.so.0.0.0
1.6M    /home/cosmicexplorer/tools/s1/opt/spack/linux-alpine3-zen3/gcc-11.2.1/subversion-1.14.1-r36gfaufu7pdi2ccsoda3cafvimlengj/lib/libsvn_fs_x-1.so.0.0.0
564K    /home/cosmicexplorer/tools/s1/opt/spack/linux-alpine3-zen3/gcc-11.2.1/subversion-1.14.1-r36gfaufu7pdi2ccsoda3cafvimlengj/lib/libsvn_delta-1.so.0.0.0
172K    /home/cosmicexplorer/tools/s1/opt/spack/linux-alpine3-zen3/gcc-11.2.1/subversion-1.14.1-r36gfaufu7pdi2ccsoda3cafvimlengj/lib/libsvn_ra_local-1.so.0.0.0
2.9M    /home/cosmicexplorer/tools/s1/opt/spack/linux-alpine3-zen3/gcc-11.2.1/subversion-1.14.1-r36gfaufu7pdi2ccsoda3cafvimlengj/lib/libsvn_wc-1.so.0.0.0
244K    /home/cosmicexplorer/tools/s1/opt/spack/linux-alpine3-zen3/gcc-11.2.1/subversion-1.14.1-r36gfaufu7pdi2ccsoda3cafvimlengj/lib/libsvn_ra-1.so.0.0.0
428K    /home/cosmicexplorer/tools/s1/opt/spack/linux-alpine3-zen3/gcc-11.2.1/subversion-1.14.1-r36gfaufu7pdi2ccsoda3cafvimlengj/lib/libsvn_diff-1.so.0.0.0
2.7M    /home/cosmicexplorer/tools/s1/opt/spack/linux-alpine3-zen3/gcc-11.2.1/subversion-1.14.1-r36gfaufu7pdi2ccsoda3cafvimlengj/lib/libsvn_subr-1.so.0.0.0
1.6M    /home/cosmicexplorer/tools/s1/opt/spack/linux-alpine3-zen3/gcc-11.2.1/subversion-1.14.1-r36gfaufu7pdi2ccsoda3cafvimlengj/lib/libsvn_fs_fs-1.so.0.0.0
```

Since `svn --help` is successfully running, and the `svn` and libraries themselves are on the order of single megabytes, I'm thinking it might make sense to avoid pulling in the rest of gettext, although it really probably won't be noticed at all either way.

So sticking with `-lintl` for now and adding a comment that `.libs.link_flags` is the canonical way to do this.